Docs/developer_guide/script_repository/markups.md

```diff
@@ -513,7 +513,7 @@ Event management of Slicer-4.11 version is still subject to change. The example
 ```python
 def onMarkupChanged(caller,event):
   markupsNode = caller
-  sliceView = markupsNode.GetAttribute("Markups.MovingInSliceView")
+  sliceView = markupsNode.GetAttribute(slicer.vtkMRMLMarkupsNode.GetMovingInSliceViewAttributeName())
```

Comment:
If we make `MovingInSliceView` information public. It could be more developer-friendly to add a `GetMovingInSliceView` (maybe also a `SetMovingInSliceView` and `ClearMovingInSliceView`) method to the markups display node.

Reply:
>  It could be more developer-friendly [...] methods

After discussing with @lassoan , we agreed that moving forward it would be sensible to relying on an dedicated API to check the status of Markups and associated control points. That said, this is a more involved task that will be done later:
* https://github.com/Slicer/Slicer/issues/8200

In the meantime, we will move forward with integrating the functions to get the attribute name instead of directly using string.